_posts/2019-07-04-BazelCon-SaveTheDate.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: posts
+title: "BazelCon 2019: Save the Date!"
+authors:
+  - dslomov
+---
+
+**TL;DR: BazelCon 2019 will be in Sunnyvale, CA on December 11-12. Please fill out 
+[this form](https://forms.gle/WqoZBfw2PWRfuLBv8) to sign up to give a talk or demo, give us feedback on topics that interest you, or both!**
+
+We’re pleased to announce that BazelCon 2019 will be held in Sunnyvale, California at Google’s Moffett Place Campus
+on December 11 and 12! ([Link to Map](https://www.google.com/maps/place/Google+Building+MP7,+Sunnyvale,+CA+94089/@37.4063379,-122.0233335,17z/data=!3m1!4b1!4m5!3m4!1s0x808fb7b9f616641b:0x19d5d6ab8d02d7!8m2!3d37.4063337!4d-122.0211448)).
+This is THE event for everything related to Bazel, the Bazel Rules Ecosystem, and fast, correct, large scale builds.
+
+Google's Event Center at Moffett Place was designed with events like BazelCon in mind, and is by far
+the largest BazelCon venue yet.  We’re excited to be able to accommodate double_ _the number of attendees
```

Comment:
The underscores look wrong.

Also, replace double with twice? (sounds better to me)

Reply:
Fixed